File: SistemaCompra/Back/src/SistemaCompra.Domain/Pedido.cs
using System;
using System.Collections.Generic;

namespace SistemaCompra.Domain
{
    public class Pedido
    {
        public int Id { get; set; }
        public int StatusAprov { get; set; }
        public DateTime DataEmissao { get; set; }
        public string Aprovador { get; set; }
        public DateTime DataAprovacao { get; set; }
        public string Observacao { get; set; }
         public int cotacaoId { get; set; }
         public Cotacao cotacao { get; set; }
         public IEnumerable<ItemPedido> itensPedidos { get; set; }

        

    }
}

File: Front/ProEventos-App/src/app/models/Pedido.ts
import { Cotacao } from './Cotacao';
import { ItemPedido } from './ItemPedido';

export interface Pedido {
 id: number;
 statusAprov: number;
 dataEmissao: string;
 aprovador: string;
 dataAprovacao: string;
 observacao: string;
 cotacaoId: number;
 cotacao: Cotacao;
 itensPedidos: ItemPedido[];
}
